common/djangoapps/philu_commands/management/commands/auto_generate_certificates_for_open_courses.py

```diff
@@ -3,23 +3,32 @@
 enrolled in currently running courses with early_no_info or early_with_info set
 in the certificate_display_behavior setting in course advanced settings
 """
-import json
+
 from logging import getLogger
 
 from django.core.management.base import BaseCommand
 
+from pytz import utc
+
 from courseware.views.views import _get_cert_data
 from lms.djangoapps.certificates.models import CertificateStatuses
 from lms.djangoapps.certificates.api import generate_user_certificates
 from student.models import CourseEnrollment
 from xmodule.modulestore.django import modulestore
+from datetime import datetime
+from django.apps import apps
```

Comment:
move them up

Reply:
any performance impact?

Reply:
btw moved!

Reply:
reason behind asking you to move them up (to group them properly) was

> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
@zeshanarifios its style guide right??